src/silx/io/url.py

```diff
@@ -37,6 +37,37 @@
 _logger = logging.getLogger(__name__)
 
 
+def slice_to_string(s):
```

Comment:
Best to keep it private:
```suggestion
def _slice_to_string(s):
```